### PR TITLE
docs: Updates and corrections to Beginner Guide

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary

Updates because of new developments and corrections of inaccuracies or errors in the Beginner Guide

### Location
Beginner Guide

### ToDo:

- [x] Starting the aircraft - APU and COVID procedures
- [x] Engine start - APU Bleed on - highlight better
- [x] Pushback using the EFB instead of or in addition to MSFS text
- [x] Descent planning using Fix Info

Maybe: 
- [x] Update screenshots to new textures

## To review:
- Changes in content as per Todo
- New images - quality and links (had some issues with upper and lower case)

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

**Comment by NUTTTTTTTT** [comment]
Really great update Mav! I love the addition of Covid-19 SOP's! Adds a really nice element of realism, and is something I'll be adding to my own SOP's in sim from now on!

Pushback section is brill, screenshots and formatting are great for readability, screenshots of the individual buttons with explanations is a really good touch, great to be encouraging use of the flyPad system. 

Checked the updated images, all display correctly on my end as far as I can tell and look great.

Not familiar with  FIX INFO as not had chance to try it out yet, however proof read and suggestions made accordingly. :)

File: docs/pilots-corner/beginner-guide/descent.md [hunk 1]
```diff
@@ -105,7 +105,26 @@ This guide will cover these phases:
 
     ![ECAM PROG page](../assets/beginner-guide/descent/ECAM-Progress-page.png "ECAM PROG page"){loading=lazy width=45% align=left}
 
-    ![ND distance rings](../assets/beginner-guide/descent/ND-disctance-rings.png "ND distance rings"){loading=lazy width=40% align=right}
+    ![ND distance rings](../assets/beginner-guide/descent/ND-disctance-rings.png "ND distance rings"){loading=lazy width=45% align=right}
+
+??? tip "FIX INFO in Experimental Version"
+    ##### Fix Info Distance Ring
+
+    Also a great aid for knowing when to descent is the new FIX INFO feature available for testing in the Experimental Version.
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/descent.md]:
```suggestion
    Also a great aid for knowing when to descend is the new FIX INFO feature available for testing in the Experimental Version.
```

Comment (Valastiri) [reply docs/pilots-corner/beginner-guide/descent.md]: Possibly add link to new guide in this admonition

File: docs/pilots-corner/beginner-guide/engine-start-taxi.md [hunk 2]
```diff
@@ -75,24 +75,38 @@ At this point we may begin pushback away from the terminal onto the taxiway Alph
 
 ---
 
-### How to Use MSFS Pushback
+### How to Pushback
 
-For the purposes of this guide we will use the default MSFS pushback controls built into ATC.
+The FlyByWire A32NX has a ground operations page on its build in flyPad EFB (Electronic Flight Bag). This page allows to control the pushback of the aircraft and other useful ground operations such as calling the jetway, baggage or catering, etc.
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/engine-start-taxi.md]:
```suggestion
The FlyByWire A32NX has a ground operations page on its built-in flyPad EFB (Electronic Flight Bag). This page allows you to control the pushback of the aircraft and other useful ground operations such as calling the jetway, baggage or catering, etc.
```

File: docs/pilots-corner/beginner-guide/engine-start-taxi.md [hunk 3]
```diff
@@ -75,24 +75,38 @@ At this point we may begin pushback away from the terminal onto the taxiway Alph
...
-1. Open the ATC menu using the toolbar at the top of your screen.
-   ![atc toolbar](../assets/beginner-guide/taxi/toolbar.png){loading=lazy}
-2. Locate `GROUND SERVICES` and select it.
-3. Select `REQUEST PUSHBACK`.
-    - You should hear your virtual pilot request pushback and ATC respond confirming. At this time the pushback tug should slowly make its way to your nose wheel.
-    - You will notice a slight "jerk" as it connects with your nose wheel. Verify `NWS STRG DISC` is listed on the ECAM memos.
-4. Release your parking brake.
-5. In the ATC window use `REQUEST PUSHBACK STEER TO THE LEFT` OR `REQUEST PUSHBACK STEER TO THE RIGHT` to turn the aircraft.
-6. Select `REQUEST PUSHBACK STOP` to end pushback when you are satisfied with your position on the taxiway.
+Although Microsoft Flight Simulator also has some pushback functionality build into the default ATC service this guide will only cover the A32NX pushback functionality.
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/engine-start-taxi.md]:
```suggestion
Although Microsoft Flight Simulator also has some pushback functionality built into the default ATC service this guide will only cover the A32NX pushback functionality.
```

File: docs/pilots-corner/beginner-guide/engine-start-taxi.md [hunk 4]
```diff
@@ -75,24 +75,38 @@ At this point we may begin pushback away from the terminal onto the taxiway Alph
...
+
+!!! block ""
+    ![Call Tug](../assets/beginner-guide/taxi/call-tug-button.png "Call Tug"){loading=lazy align=left}
+
+    After we received clearance to pushback we will call the pushback tug by pressing the `Call Tug` button on the flyPad.
+    
+    If a pushback tug is available at this gate or stand it will then start attaching itself to the nose wheel.
+
+    !!! warning ""
+        Some airports / gates / stands do not show a tug. This functionality still works and you can push back as if a tug is attached. I looks like an invisible tug is pushing the aircraft.
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/engine-start-taxi.md]:
```suggestion
        Some airports / gates / stands do not show a tug. This functionality still works and you can push back as if a tug is attached. It looks like an invisible tug is pushing the aircraft.
```

File: docs/pilots-corner/beginner-guide/landing.md [hunk 5]
```diff
@@ -86,10 +85,13 @@ To intercept the ILS Localizer we follow these steps:
...
 
-    ![Activate LOC mode on FCU](../assets/beginner-guide/landing/FCU-ILS-APPR.png "Activate APPR mode on FCU"){loading=lazy}
+    ![Activate APPR mode on FCU](../assets/beginner-guide/landing/FCU-ILS-APPR.png "Activate APPR mode on FCU"){loading=lazy}
 
     ![ILS lateral deviation marker moving inwards](../assets/beginner-guide/landing/PFD-ILS-deviation-scale.png "ILS lateral deviation marker moving inwards"){loading=lazy}
 
+??? note "What is the blue dashed line?"
+    You might ask what the blue dashed line is. It is part of the new FIX INFO feature available only in the Experimental version for the time being. It allows to draw distance rings or radial lines from navigation fixes. For this illustration it helped us to visualize the localizer signal path at 261° into the runway which we are going to capture with the APPR mode.
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/landing.md]:
```suggestion
    You might ask what the blue dashed line is. It is part of the new FIX INFO feature available only in the Experimental version for the time being. It allows to draw distance rings or radial lines from navigation fixes. For this illustration it helps us to visualize the localizer signal path at 261° into the runway which we are going to capture with the APPR mode.
```

File: docs/pilots-corner/beginner-guide/starting-the-aircraft.md [hunk 6]
```diff
@@ -31,100 +31,148 @@ Aircraft is in a cold and dark state at a gate.
...
+??? note "Real World APU and A/C Start Procedure"
+    The above process is a simplified version of how real world airline start up the APU (documented in their SOP - Standard Operating Procedures).
 
-    ![APU Start](../assets/beginner-guide/after-landing/APU-Start.png "APU Start"){loading=lazy width=25%}
+    Here is a description of an actual real world procedure of a well known airline.
+
+    - Check A/C Packs are OFF before powering the aircraft
+    - Set APU MASTER switch to ON
+    - Press APU START 
+        - some airlines still require the pilot to wait 3sec for the APU flap to open. This is no longer required in the A320neo as the startup sequence waits for the flap to be open automatically.
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/starting-the-aircraft.md]:
```suggestion
        - Some airlines still require the pilot to wait 3sec for the APU flap to open. This is no longer required in the A320neo as the startup sequence waits for the flap to be open automatically. 
```

**Review by Valastiri — CHANGES_REQUESTED** [review]
See comment in Discord about the force-push and rebase to remove unnecessary commits from `primary`

**Review by Valastiri — APPROVED** [review]

**Pull request merged**